src/app/common/crypto-assets/stacks-crypto-asset.utils.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { StacksFungibleTokenAsset } from '@shared/models/crypto-asset.model';
+
+import {
+  isFtNameLikeStx,
+  isIconUrl,
+  isTransferableStacksFungibleTokenAsset,
+} from './stacks-crypto-asset.utils';
+
+describe(isFtNameLikeStx.name, () => {
+  it('detect impersonating token names', () => {
+    expect(isFtNameLikeStx('STX')).toBeTruthy();
+    expect(isFtNameLikeStx('stx')).toBeTruthy();
+    expect(isFtNameLikeStx('stacks')).toBeTruthy();
+    expect(isFtNameLikeStx('Stäcks')).toBeTruthy();
+    expect(isFtNameLikeStx('Stácks')).toBeTruthy();
+    expect(isFtNameLikeStx('Stáçks')).toBeTruthy();
+    expect(isFtNameLikeStx('stocks')).toBeFalsy();
+    expect(isFtNameLikeStx('miamicoin')).toBeFalsy();
+    expect(isFtNameLikeStx('')).toBeFalsy();
+  });
+});
+
+describe(isIconUrl.name, () => {
```

Comment:
The rationale is that, a `describe` block often annotates that a given group of tests relate to that function. 

I've seen tests called something completely different, because someone renamed the function, without renaming the test.

Good spot on the variable declaration, though this looks like a problem with the code:

```ts
export const isIconUrl = isValidUrl;
```
There's no reason to do this ☝🏼 

- There are already tests written for `isValidUrl` ([here](src/shared/utils/validate-urls.spec.ts))
- `isIconUrl` is confusing because it doesn't check if it's an icon url at all... it checks if it's a valid url, and it should be called directly in the icon code